GenderPayGap.WebUI/wwwroot/styles/components/_number-icons.scss

```diff
@@ -0,0 +1,28 @@
+﻿.action-steps {
+  position: relative;
+  padding-bottom: 10px;
+  padding-left: -30px;
+
+  ol {
+    list-style: none;
+    counter-reset: action-step-counter;
+    padding-left: 0;
+  }
+  ol > li {
+    counter-increment: action-step-counter;
+  }
+  ol > li::before {
+    content: counter(action-step-counter);
+    position: absolute;
+    line-height: 23px;
+    font-size: small;
+    text-align: center;
+    vertical-align: middle;
+    width: 23px;
+    height: 23px;
+    left: -30px;
+    color: white;
+    background: black;
+    border-radius: 50%;
+  }

```

Comment:
This is all _very cool_ 😎 but I wonder if it would be easier to read & more in-keeping with Design System styling if we use a more normal number styling.

e.g. just making the numbers bold:
```scss
ol > li::marker {
    font-weight: bold;
}
```

If you'd prefer to keep this styling, there's a small bug with how it looks on mobile (the numbers fall off the left of the screen)

Reply:
I thought about this - but a lot of the guidance has been created with various versions of these icons (here they're numbers, on the new version there were replacement image icons) so I wasn't sure if it was likely that we'd want to keep this styling in the case of an eventual redesign? Happy to go back to just bold numbers, but maybe for now it's a question for Claire r.e. style since it won't make much difference technically?

Reply:
Yeh sure - happy to leave them as numbers in circles.
We can present the "just bold numbers" as an alternative to Claire when she looks at this in future.
It's good that we know how to do this either way.

There's a remaining small bug on narrow screens (the numbers fall off the left of the screen) - I expect some margin/padding somewhere could fix this though

Reply:
(I've fixed the numbers falling off the page on mobile for now 😄)